pages/national_resources/gr_resources.md

```diff
@@ -31,8 +31,8 @@ ref_to_main_resources:
 
 # List here tools and resources mainly relevant for the specific country
 national_resources: 
-  - name: <!---REPLACE THIS with the national resources about RDM in life sciences such as local instances of tools, guidelines or regulations--->
-    description:
+  - name: RDMkit
```

Comment:
is this just a draft for now? because RDMkit is not a national tool/resource. Happy to review the page once first draft is ready :) Thank you @eleni-adamidi

Reply:
Hey @Nazeeefa yes, this was just a draft to test that everything goes in the right place. Will add the relative content so that you can review once ready, thanks!